mb_modelbase/cache/MemcachedCache.py

```diff
@@ -0,0 +1,29 @@
+from mb_modelbase.cache import BaseCache
+from pymemcache.client import base
+
+class MemcachedCache(BaseCache):
+    """
+    A cache with memcached as storage backend
+
+    Attributes:
+        hostname (str): hostname of memcached server
+        port (int): port of memcached server
```

Comment:
Why is hostname and port listed as attributes of the class if it really is paramters to the constructor?

Reply:
Fixed!